### PR TITLE
Add a Contributing.md 

### PR DESCRIPTION
**Pull request by georgehrke**
Hopefully this will encourage more people to use the issue template when creating issues.

please review @RealRancor @jancborchardt

### CODE REVIEW

**Comment by georgehrke** [comment]
let me add a faq

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Help us to maximize the effort we can spend fixing issues and adding new features, by not reporting duplicate issues.
+
+[template]: https://raw.github.com/owncloud/core/master/issue_template.md
+[mailinglist]: https://mailman.owncloud.org/mailman/listinfo/owncloud
+[forum]: https://forum.owncloud.org/
+[irc]: https://webchat.freenode.net/?channels=owncloud&uio=d4
+
+### FAQ
+*  I can't hide shared calendars / birthday calendar - see https://github.com/owncloud/calendar/issues/21
+* I can't delete a single occurrence of a repeating event - see https://github.com/owncloud/calendar/issues/33
```
Comment (ghost) [inline CONTRIBUTING.md:27]: - I can't publish a public calendar - see https://github.com/owncloud/calendar/issues/245
- I can't subscribe to an external calendar - see https://github.com/owncloud/calendar/issues/192

**Comment by ghost** [comment]
Except from the links above :+1:

**Comment by raghunayyar** [comment]
I think this works. Merging and closing.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+* This repository ([core](https://github.com/owncloud/calendar/issues)) is *only* for issues within the ownCloud calendar code.
+* __SECURITY__: Report any potential security bug to security@owncloud.com following our [security policy](https://owncloud.org/security/) instead of filing an issue in our bug tracker
+* Report the issue using our [template][template], it includes all the information we need to track down the issue.
+
+Help us to maximize the effort we can spend fixing issues and adding new features, by not reporting duplicate issues.
+
+[template]: https://raw.github.com/owncloud/core/master/issue_template.md
+[mailinglist]: https://mailman.owncloud.org/mailman/listinfo/owncloud
+[forum]: https://forum.owncloud.org/
+[irc]: https://webchat.freenode.net/?channels=owncloud&uio=d4
```
Comment (jancborchardt) [inline CONTRIBUTING.md:23]: Wrong links here. Let’s not advertise the mailing list nor the forum. Only this issue tracker here and our own IRC channel #owncloud-calendar is relevant.

Comment (georgehrke) [inline CONTRIBUTING.md:23]: but #owncloud-calendar is a dev channel and not for user support, we should add link to #ownCloud-calendar in the "Contribute Code and translations" section, but not up here

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+### Short version
+
+ * The [**issue template can be found here**][template]. Please always use the issue template when reporting issues.
+
+### Guidelines
+* Please search the existing issues first, it's likely that your issue was already reported or even fixed.
+  - Go to one of the repositories, click "issues" and type any word in the top search/command bar.
+  - You can also filter by appending e. g. "state:open" to the search string.
+  - More info on [search syntax within github](https://help.github.com/articles/searching-issues)
+* This repository ([core](https://github.com/owncloud/calendar/issues)) is *only* for issues within the ownCloud calendar code.
```
Comment (ghost) [inline CONTRIBUTING.md:14]: Sorry, had overlooked this. [core] should be [calendar]

**Pull request merged**